lib/index.ts

```diff
@@ -45,7 +45,16 @@ interface Options {
   args?: string[];
 }
 
-export async function inspect(root, targetFile, options: Options = {}) {
+export interface GoDepsConfiguration {
+  includeGoStandardLibraryDeps?: boolean;
+}
+
+export async function inspect(
+  root,
+  targetFile,
+  configuration: GoDepsConfiguration,
```

Comment:
I think the configuration should be part of options, so that we follow the interface here: https://github.com/snyk/snyk-cli-interface/blob/master/legacy/plugin.ts#L17